tests/TableViewTest.cc

```diff
@@ -58,7 +58,7 @@ TEST(TableViewTest, testCreateTableView) {
     // Test async create and close the client during the process.
     Latch latch(1);
     client.createTableViewAsync(
-        topic, tableViewConfiguration, [&latch](Result result, const TableView& tableView) {
+        topic, {.schemaInfo = schemaInfo}, [&latch](Result result, const TableView& tableView) {
```

Comment:
Why is it flaky when using the incompatible schema? It should always fail but it didn't,

Reply:
Because this is to close the tableView during asynchronous creation. 

I thought about it for a moment, and it is also problematic to modify it like this. It is possible to receive `result == ResultOk`

This test is intended to cover that it is possible to close during asynchronous creation. But this can be a lot of cases, so it does not make sense. So I removed that section of the test